doc/develop/backend-api.rst

```diff
@@ -237,20 +242,46 @@ a single component. That is why Scanner inherit from Emitter.
     The following three attributes permit to define a region of interest
     (i.e., a sub-region).
 
-    .. py:attribute:: translation
+    .. py:attribute:: scale
 
-        *(VA, tuple of floats, unit=px)* How much shift is applied to the center of the area acquired. It is expressed in pixels (the size of a pixel being defined by pixelSize, and so independent of .scale).
+        *(VA, tuple of floats or int, unit=ratio)* ratio of the size of the scannable area divided by the size of the scanned area.
+        By convention, (1, 1) corresponds to the closest distance between pixels centers while scanning at the highest resolution.
+        Note that this is the inverse of the typical definition of scale (i.e., increasing the scale leads to a smaller scanned area).
+        The advantage of this definition is that its meaning is very similar to binning.
+        Note that the MD_PIXEL_SIZE metadata of a dataflow will depend both on pixelSize and scale (i.e., MD_PIXEL_SIZE = pixelSize * scale).
 
-    .. py:attribute:: scale
+    .. py:attribute:: translation
 
-        *(VA, tuple of floats or int, unit=ratio)* ratio of the size of the scannable area divided by the size of the scanned area. Note that this is the inverse of the typical definition of scale (i.e., increasing the scale leads to a smaller scanned area). The advantage of this definition is that its meaning is very similar to binning. Note that the MD_PIXEL_SIZE metadata of a dataflow will depend both on pixelSize and scale (i.e., MD_PIXEL_SIZE = pixelSize * scale).
+        *(VA, tuple of floats, unit=px)* How much shift is applied to the center of the area acquired.
+        It is expressed in pixels (the size of a pixel being defined by pixelSize, and so independent of .scale).
+        It is limited by the field of view of the scanner, and it is automatically
+        clipped based on the current resolution and scale.
+        A bigger value in X means a shift towards the right, and a bigger value in Y means a shift towards the bottom.
```

Comment:
For the image perspective this is correct. Physically the beam moves in opposite directions. Maybe be something to clear?